src/react/components/toolbar/toolbarItem.tsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
-import { IProject } from "../../../models/applicationState";
+import { IProject, RegionType, EditorMode } from "../../../models/applicationState";
```

Comment:
If we make some of the changes mentioned above we likely don't need any of these changes.